middleware/logmiddleware.go

```diff
@@ -93,19 +94,32 @@ func logBeforeHandler(ctx loggingContext) {
 	}).Trace("incoming request")
 }
 
-func logAfterHandler(ctx loggingContext, startTime time.Time) {
-	glogger.Get(ctx.Context()).WithFields(logrus.Fields{
+func logAfterHandler(ctx loggingContext, startTime time.Time, fiberErr *fiber.Error) {
```

Comment:
Since this function is not framework specific, I'd accept a generic `error` interface and then do type assertion at line 107 if needed

Reply:
I agree, done